style-guide/event.html

```diff
@@ -0,0 +1,232 @@
+	<link href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,900,900i" rel="stylesheet">
+	<script src="http://news.wsu.edu/wp-includes/js/jquery/jquery.js?ver=1.12.4" type="text/javascript">
+	</script>
+	<script src="http://news.wsu.edu/wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1" type="text/javascript">
+	</script>
+	<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js?ver=4.8.1" type="text/javascript">
+	</script>
+	<script type="text/javascript">
+	       /* <![CDATA[ */
+	       var spineoptions = {
+	           "social": {
+	               "share_text": "First Post | Internal News | Washington State University",
+	               "twitter_text": "First Post",
+	               "twitter_handle": "wsupullman"
+	           }
+	       };
+	       /* ]]> */
+	</script>
+	<script src="https://repo.wsu.edu/spine/1/spine.min.js?ver=0.27.14-1.6.6-41211" type="text/javascript">
+	</script>
+	<style type="text/css">
+	       .recentcomments a {
+	           display: inline !important;
+	           padding: 0 !important;
+	           margin: 0 !important;
+	       }
+	</style><!-- COMPATIBILITY -->
+	<!--[if lt IE 9]><script src="https://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
+	 <noscript>
+	<style>
+	           #spine #spine-sitenav ul ul li {
+	               display: block !important;
+	           }
+	</style></noscript>
+</head>
+<body class="post-template-default single single-post postid-1 single-format-standard domain-wp-wsu-dev path-news-internal categorized-uncategorized depth-4 section-2017 sub-section-09 sub-sub-section-22 sub-sub-sub-section-first-post page-first-post">
+	<a class="screen-reader-shortcut" href="#wsuwp-main">Skip to main content</a> <a class="screen-reader-shortcut" href="#spine-sitenav">Skip to navigation</a>
+	<div class="style-skeletal colors-default spacing-default" id="jacket">
+		<div class="fluid folio max-1386" id="binder">
+			<header class="site-header">
+				<div class="site-title">
+					WSU Events
+				</div>
+			</header>
+			<main id="wsuwp-main">
+				<article class="post type-post status-publish hentry card-single card--event">
+					<header class="card-header">
+						<p class="card-event-type">Seminar</p>
+						<h1 class="card-title">WSU Tri-Cities celebrates grand opening of student union building</h1>
+					</header>
+					<section class="card-details row halves">
+            <div class="column one">
+  						<time class="card-event-datetime" datetime="2017-11-6T19:30"><span class="card-date">Monday, November 6, 2017 - <a class="card-calendar-add" href="#">add to calendar</a></span> <span class="card-time">7:30 pm</span></time>
+  						<div class="card-location">
+  							<p class="card-address">The Torch Club - 18 Waverly Place, New York, NY 10003</p>
+  							<p class="card-location-notes">Go there and then you need to take a slight left up the stairs.</p>
```

Comment:
I almost wonder if `card-location-notes` can be removed. Any `<p>` without a class can be styled as `card-location p`, which would allow for multiple generic paragraphs if needed. Specific paragraphs like `card-address` could still be used.